explainer.md

```diff
@@ -99,68 +99,153 @@ This aim of this API is to help websites provide an optimal initial experience.
+ 1. `rMKSA` already does this. The returned `MediaKeySystemAccess` contains a `MediaKeySystemConfiguration` that describes a sequence of key-system-permitted media mime types and robustnesses. The `rMSKA` algorithm produces this sequence by starting with the input sequence and removing incompatible options (prioritizing those that come earlier in the list). Incompatibility may arise when a given codec or robustness requires a particular pipeline (e.g. hw secure decrypt+decode) which may not support a codec or robustness described later in the input sequence. The media keys used by this pipeline may be impossible to export, blocking transitions to another pipeline with different capabilities. 
 
-#### Building on top of the Permissions API
+    If MediaCapabilities is to stand in for `rMKSA`, it should similarly be capable describing the restricted set of codecs/robustnesses allowed by a given key system configuration.
 
-This solution is very similar to the previous one but instead of exposing the permission status in the Media Capabilities, it is re-using other components of the Web Platform. With this approach, the developer would have to query the Permissions API for the permissions status and the Media Capabilities API for the capabilities. The capabilities will matching the current permission status. As above, the capabilities should probably expose the not granted results if the permission status is set to `prompt`. In addition of re-using other APIs, this approach could also use the work in progress `request` method of the Permissions API which will allow the developers to request permission for a specific EME configuration before making a call to the `requestMediaKeySystemAccess`.
+2. Various proposals are being considered for MSE and EME to allow codec/container transitions within a MSE SourceBuffer.
+ 
+    https://github.com/wolenetz/media-source/blob/codec-switching/codec-switching-explainer.md
+    https://github.com/w3c/encrypted-media/pull/374 
+    https://github.com/w3c/encrypted-media/issues/251 
+
+
+    To use this feature effectively, developers will need to know up front (before downloading streams) whether a given codec transition can be supported. 
+
+In support of the above, Media Capabilities could describe compatible codecs by allowing chained `compatibleDecodingInfo()` calls to be issued from a previously returned `MediaCapabilitiesInfo` object. Like `decodingInfo()`, this API takes a `MediaDecodingConfiguration` dictionary and returns a `MediaCapabilitiesInfo` object. 
```

Comment:
The explainer mentions transitions at some point. Should we just point to that and say that it would be another good use case for using this without going into more details?